Firestore/Source/Local/FSTMemoryRemoteDocumentCache.mm

```diff
@@ -28,6 +30,17 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/**
+ * Returns the number of bytes in the given document key.
+ */
+static size_t FSTDocumentKeyByteSize(FSTDocumentKey *key) {
+  size_t count = 0;
+  for (auto it = key.path.begin(); it != key.path.end(); it++) {
+    count += (*it).size();
```

Comment:
How exact is this meant to be? At the very least it's excluding the path separators.

Reply:
I've updated the comment to reflect that this is an estimate. It's for memory persistence, so it doesn't have to be exact, and furthermore I don't think we actually store the separators in memory? I believe it's a vector of segments, or something like that.